src/stan/math/prim/mat/prob/multi_normal_rng.hpp

```diff
@@ -20,10 +20,10 @@ namespace stan {
   namespace prob {
     using Eigen::Dynamic;
 
-    template <class RNG>
+    template <class RNG, typename T1, typename T2>
```

Comment:
To follow Eigen's naming conventions, these should probably
be Derived1 and Derived2. 

- Bob

> On Mar 12, 2015, at 8:47 PM, Joshua Pritikin <notifications@github.com> wrote:
> 
> In src/stan/math/prim/mat/prob/multi_normal_rng.hpp:
> 
> > @@ -20,10 +20,10 @@ namespace stan {
> >    namespace prob {
> >      using Eigen::Dynamic;
> >  
> > -    template <class RNG>
> > +    template <class RNG, typename T1, typename T2>
> 
> But T1 and T2 are not scalar types to begin with. You can't constrain the scalar type using the template when you are using Eigen::MatrixBase<>
> 
> —
> Reply to this email directly or view it on GitHub.
>